<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:130}
QTabBar::tab{height:60}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>play game</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文新魏</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择难度</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>easy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lunatic</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_to_mainscene">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>仓库</string>
     </attribute>
     <widget class="QLabel" name="label_has_reborn_coin">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>label_has_reborn_coin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_missle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>导弹(已拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_laser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>激光(已拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_shield">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>护盾(已拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_screenclear">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>清屏(已拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_coins">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>label_has_coins</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_missle_no">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>导弹(未拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_laser_no">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>激光(未拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_shield_no">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>护盾(未拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_has_screenclear_no">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>430</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>清屏(未拥有)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_skill">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>技能拥有情况:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_store">
     <attribute name="title">
      <string>商店</string>
     </attribute>
     <widget class="QPushButton" name="btn_skill_missle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>导弹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_skill_laser">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>激光</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_skill_shield">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>护盾</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_skill_screenclear">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>清屏</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_value_of_coins">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>购买小技能</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>购买大招</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reborn_coin">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>购买复活币</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>购买复活币</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>排行榜</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>游戏设置</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>页</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
